src/server.h

```diff
@@ -1480,6 +1480,7 @@ struct redisServer {
     int sentinel_mode;          /* True if this instance is a Sentinel. */
     size_t initial_memory_usage; /* Bytes used after initialization. */
     int always_show_logo;       /* Show logo even for non-stdout logging. */
+    int enable_motd;            /* Flag to retrieve the Message of today using CURL request*/
```

Comment:
As this is an immutable config it should go in redisServerConst.  This makes life a little easier reviewing globals for thread safety.